Transparent CK Editor Panel/readme.mkd

```diff
@@ -0,0 +1,10 @@
+# Transparent CK Editor Panel
+The Transparent CK Editor Panel mashup make floating CK Editor panel transparent to help see elements behind it.
+
+For example it helps to see previous Test Step while editing another one.
+
+![Transparent CK Editor Panel in Test Steps](https://github.com/kruglik-alexey/TP3MashupLibrary/blob/feature/us97121_transparent_cke_panel/Transparent%20CK%20Editor%20Panel/transaprent.png)
```

Comment:
set path relative to merged mashup, for example as here https://raw.githubusercontent.com/TargetProcess/TP3MashupLibrary/master/Dark%20Theme/README.mkd